### DISCUSSION TITLE
Rubric used in choosing a CI system

### DISCUSSION THREAD

**Discussion by hokadiri** [MobileNativeFoundation/discussions#143] [General]
Hey folks!

For folks who have had the experience of changing/picking a CI tool, what rubric did you use to compare the different solutions out there?

**Comment by changusmc** [comment]
This is the rubric that I compiled when Dropbox Mobile selected a vendor last year.

[Rubric.pdf](https://github.com/MobileNativeFoundation/discussions/files/8383126/Untitled.pdf)

**Comment by valeraz** [comment]
Here is ours.

[Mobile CI System Evaluation Criteria (Template).csv](https://github.com/MobileNativeFoundation/discussions/files/8383975/Mobile.CI.System.Evaluation.Criteria.Template.csv)
